src/view/pages/statement/components/SimilarStatementsSuggestion/StepOneStatementInput.tsx

```diff
@@ -131,6 +130,22 @@ export default function StepOneStatementInput({
 							iconBackgroundColor='var(--dark-blue)'
 							onClick={handleSubmit}
 						/>
+					</div> */}
+					<div className='similarities__buttonBox'>
+						<SubmitStatementButton
+							icon={SendIcon}
+							text='Submit Statement'
+							buttonRadius="2rem 0px 0px 2rem"
+							buttonBackground="var(--button-blue)"
+							buttonWidth="10rem"
+							circlePos="relative"
+							circleRight="1rem"
+							circleBorder="2px solid var(--white)"
+							circleRadius="2rem"
+							circleBackground="var(--button-blue)"
+							textColor='var(--white)'
+							onClick={handleSubmit}
+						/>
```

Comment:
Could you reduce the number of props you pass down to the button? Some things can be by default.
The button width should be responsive with a maximum width.